docs/testing/uninstall-add-in.md

```diff
@@ -0,0 +1,142 @@
+ms.date: 12/28/2024
+ms.localizationpriority: medium
+---
+
+# Uninstall add-ins under development
+
+Incompletely removed add-ins under development can leave artifacts, such as custom ribbon buttons or registry entries, on your development computer. In the case of Outlook add-ins, these artifacts can be added to other computers when you sign into Outlook on them with the same ID as you used to develop the add-in. In this article, we call these ghost add-ins. 
+
+This article provides some guidance to minimize the chance of these problems and for resolving them if they occur.
+
+## Prevent the problems
+
+When an add-in is sideloaded, several things happen:
+
+- A web server, usually on localhost, is started to serve the add-ins files (HTML, CSS, JavaScript, etc.).
+- These same files are cached on your development computer.
+- The add-in is registered with the development computer. The registration is done with Registry entries on a Windows computer or with certain files saved to the file system on a Mac.
+- Most tools for sideloading add-ins also automatically open the Office application that the add-in targets and populate the application with any custom ribbon buttons or context menu items that are defined in the add-in's manifest.
+- For an Outlook add-in, the add-in's manifest and certain other values about the add-in are registered with the Exchange service.
+
+   > [!IMPORTANT]
+   > When you sign into Outlook, it downloads from Exchange, and sideloads, all the Outlook add-in manifests that are associated with your ID, *including add-ins that you are developing on a different computer using the same ID*. For example, any custom ribbon buttons defined in the manifest will appear for the add-in. 
+   >
+   > If the URLs in the manifest point to a non-localhost server and that server is running and accessible to the non-development computer, then Outlook caches the add-in's files in the local file system and the add-in usually runs normally on the computer. Otherwise, the add-in doesn't function, but visible parts of it, such as custom ribbon buttons appear. They have the labels defined in the manifest. The add-in's button icons also appear if they were ever cached locally on the non-development computer and the cache was never cleared. But the icon files aren't stored with Exchange, so if they were never cached on the non-development computer (or the cache has been cleared), then the buttons have default icons.
+   >
+   > Until the add-in's registration is removed from Exchange, the add-in will continue to appear. See [Remove a ghost add-in](#remove-a-ghost-add-in) for information about removing the registration in Exchange.
+
+### Use your tool's uninstall facility
+
+To prevent ghost add-ins, end every testing/debugging/sideloading session by using the uninstall (also called unacquire) facility that is provided by the tool that you used to start the session. Doing this reverses the effects in the bullet list earlier in this article.
```

Comment:
Suggestion for localization. Also, since there are multiple bullet lists before this paragraph, consider being more specific. 
```suggestion
To prevent ghost add-ins, end every testing, debugging, and sideloading session by using the uninstall (also called unacquire) option that is provided by the tool that you used to start the session. Doing this reverses the effects of sideloading, as stated earlier in this article.
```